scripts/src/analytics/events/earn_transaction_broadcast_to_blockchain_succeeded.ts

```diff
@@ -0,0 +1,31 @@
+import { Event, EventData } from "../index";
+import { Common } from "./common";
+
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+/**
+ * Server submits earn transaction to blockchain and get confirmation
+ */
+export interface EarnTransactionBroadcastToBlockchainSucceeded extends EventData {
+	event_name: "earn_transaction_broadcast_to_blockchain_succeeded";
+	event_type: "log";
+	common: Common;
+	transaction_id: string;
+	offer_id: string;
+	order_id: string;
+}
+
+export function create(user_id: string, transaction_id: string, offer_id: string, order_id: string): Event<EarnTransactionBroadcastToBlockchainSucceeded> {
```

Comment:
@doodyparizada 
because this interface is being generated based on the json schema.
i prefer to change the generated code as little as possible.

also, this is just a data object, interfaces are better for that than classes.

Reply:
Oh, you have no control over the results of this? can you add the command line that generates these files?